engine/language-server/src/main/scala/org/enso/languageserver/runtime/ContextRegistryProtocol.scala

```diff
@@ -156,15 +156,15 @@ object ContextRegistryProtocol {
     *
     * @param expressionId the id of updated expression
     * @param `type` the updated type of expression
-    * @param methodPointer the suggestion id of the updated method pointer
+    * @param methodPointer the updated method pointer
     * @param profilingInfo profiling information about the expression
     * @param fromCache whether or not the expression's value came from the cache
     * @param payload an extra information about the computed value
     */
   case class ExpressionUpdate(
     expressionId: UUID,
     `type`: Option[String],
-    methodPointer: Option[Long],
+    methodPointer: Option[MethodPointer],
```

Comment:
This will probably increase the size of `ExpressionUpdate` messages - instead of a number we send the tripple with three strings, right?